### ISSUE TITLE
question about batch norm position

### ISSUE THREAD

**Issue by jlxAtNovozymes** [skeydan/Deep-Learning-and-Scientific-Computing-with-R-torch#9]
Hi, when I read chapter 18.3 https://skeydan.github.io/Deep-Learning-and-Scientific-Computing-with-R-torch/image_classification_2.html#run-2-batch-normalization.

I noticed that in your code, some batch norm layers are after activation layer, some batch norm layers are before activation layer. could you help me explain a litter further
![image](https://github.com/skeydan/Deep-Learning-and-Scientific-Computing-with-R-torch/assets/90908083/c746ced8-78dc-4780-8589-b48347bd12b3)

**Comment by skeydan** [comment]
Hi,

please understand that I cannot answer questions about deep learning in this repository. 
I'd recommend asking in Posit community forum, where there are lots of friendly users around :-)

https://community.rstudio.com/c/ml/15
